docs/guides/composition-revisions-example.md

```diff
@@ -0,0 +1,302 @@
+``` 
+
+#### Verify the updated Composition revisions
+Verify that Crossplane creates a new Composition revision:
+```shell
+kubectl get compositionrevisions -o=custom-columns=NAME:.metadata.name,REVISION:.spec.revision,LABEL:.metadata.labels.channel
+```
+Expected Output:
+```shell
+NAME                                    REVISION   LABEL
+myvpcs.aws.example.upbound.io-727b3c8   2          staging
+myvpcs.aws.example.upbound.io-ad265bc   1          dev
+``` 
+
+Crossplane assigns the Composite Resources `vpc-auto` and `vpc-staging` to Composite revision:2.  
+XRs `vpc-man` and `vpc-dev` are still assigned to the original revision:1:
+
+```shell
+kubectl get composite -o=custom-columns=NAME:.metadata.name,SYNCED:.status.conditions[0].status,REVISION:.spec.compositionRevisionRef.name,POLICY:.spec.compositionUpdatePolicy,MATCHLABEL:.spec.compositionRevisionSelector.matchLabels
+```
+Expected Output:
+```shell
+NAME          SYNCED   REVISION                                POLICY      MATCHLABEL
+vpc-auto      True     myvpcs.aws.example.upbound.io-727b3c8   Automatic   <none>
+vpc-dev       True     myvpcs.aws.example.upbound.io-ad265bc   Automatic   map[channel:dev]
+vpc-man       True     myvpcs.aws.example.upbound.io-ad265bc   Manual      <none>
+vpc-staging   True     myvpcs.aws.example.upbound.io-727b3c8   Automatic   map[channel:staging]
+``` 
+
+{{< hint "note" >}}
+`vpc-auto` always use the latest Revision.  
+`vpc-staging` now matches the label applied to Revision revision:2.
+{{< /hint >}}
+
+### Enable DNS support in the Composition
+Update the Composition to disable DNS support in the VPC and change the label from `staging` back to `dev`.
+
+Apply the following changes to update the `Composition` spec and label:
+```yaml
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+  labels:
+    channel: dev
+  name: myvpcs.aws.example.upbound.io
+spec:
+  writeConnectionSecretsToNamespace: crossplane-system
+  compositeTypeRef:
+    apiVersion: aws.example.upbound.io/v1alpha1
+    kind: MyVPC
+  resources:
+  - base:
+      apiVersion: ec2.aws.upbound.io/v1beta1
+      kind: VPC
+      spec:
+        forProvider:
+          region: us-west-1
+          cidrBlock: 192.168.0.0/16
+          enableDnsSupport: false
+          enableDnsHostnames: true
+    name: my-vcp
+```
+
+Expected Output:
+```shell
+composition.apiextensions.crossplane.io/myvpcs.aws.example.upbound.io configured
+``` 
+
+#### Verify the updated Composition revisions
+Verify that Crossplane creates a new Composition revision:
+
+```shell 
+kubectl get compositionrevisions -o=custom-columns=NAME:.metadata.name,REVISION:.spec.revision,LABEL:.metadata.labels.channel
+```
+Expected Output:
+```shell
+NAME                                    REVISION   LABEL
+myvpcs.aws.example.upbound.io-727b3c8   2          staging
+myvpcs.aws.example.upbound.io-ad265bc   1          dev
+myvpcs.aws.example.upbound.io-f81c553   3          dev
+``` 
+
```

Comment:
What happened above? A short explanation would be helpful.
The explanation below should be moved after user see output of the next command.

Reply:
done